modules/multi-architecture-scheduling-runtimeclass.adoc

```diff
@@ -0,0 +1,60 @@
+// module included in the following assembly 
```

Comment:
I would get rid of this since it's overkilling. It requires the users to set (and know about) the runtimeclass `handler`. It seems that runtimeclasses are meant specifically to change the handler and their scheduling fields are there for supporting this main feature.

Windows containers users must use this API, multi-arch ones don't need it

Reply:
agreed. let's remove runtimeclass and its examples. It was relevant for  windows, but not here.